src/assets/scripts/client/airportGuide/AirportGuideView.js

```diff
@@ -0,0 +1,135 @@
+import { SELECTORS } from '../constants/selectors';
+
+/**
+ * Display information about the active airport
+ *
+ * Part of the build includes copying and converting source
+ * documents to html. This is retrieved via ajax request and
+ * stored in memory until needed
+ *
+ * _Original documentation can be found in `documentation/airport-guides`_
+ *
+ * @class AirportGuideView
+ */
+export default class AirportGuideView {
+    /**
+     * @constructor
+     * @param {JQuery|HTMLElement} $element
+     * @param {string} data  html string generated from airport-guide markdown
+     */
+    constructor($element, data) {
+        /**
+         * The HTML view container of the data (formatted)
+         *
+         * @property $element
+         * @type {JQuery|HTMLElement}
+         * @default null
+         */
+        this.$element = null;
+
+        /**
+         * Local instance of the airport guide data
+         *
+         * @property _airportGuideMarkup
+         * @type {string}
+         */
+        this._airportGuideMarkup = data;
+
+        /**
+         * The HTML containing the data itself
+         *
+         * @property $data
+         * @type {JQuery|HTMLElement}
+         */
+        this.$airportGuideView = null;
+
+        return this._init()
+            ._createChildren($element)
+            .enable();
+    }
+
+    /**
+     * Lifecycle method
+     *
+     * Should be called once on instantiation
+     *
+     * @for AirportGuideView
+     * @method _init
+     * @private
+     * @chainable
+     */
+    _init() {
```

Comment:
@n8rzz  shoudn't this be public?

Reply:
ahh, damn, I'm breaking one of my conventions.  fixed